packages/edge-bundler/test/integration/test.js

```diff
@@ -8,7 +8,7 @@ import { fileURLToPath, pathToFileURL } from 'url'
 import { promisify } from 'util'
 
 import cpy from 'cpy'
-import tar from 'tar'
+import { extract } from 'tar'
```

Comment:
This is because I had initially upgraded tar but I [had to back out](https://github.com/isaacs/node-tar/pull/435). The current version has named exports so I just left this in.